code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -185,8 +185,18 @@
 				m_type = VISIBLE
 			else
 				if (!muzzled)
-					message = "<B>[src]</B> coughs!"
-					m_type = HEARABLE
+					if (auto == 1)
+						if(world.time-lastScream >= 30)//prevent cough spam
```

Comment:
If you're going to go for this lastScream overlap, you should rename it.
last_emote_noise, maybe.